src/main/java/edu/harvard/iq/dataverse/dataverse/banners/NewBannerPage.java

```diff
@@ -68,6 +71,19 @@ public String init() {
         return StringUtils.EMPTY;
     }
 
+    public void uploadFileEvent(FileUploadEvent event) {
+        String locale = (String) event.getComponent().getAttributes()
+                .get("imageLocale");
+
+        dto.getDataverseLocalizedBanner().stream()
+                .filter(dlb -> dlb.getLocale().equals(locale))
+                .forEach(dlb -> {
+                    dlb.setFile(event.getFile());
+                    dlb.setUploadedDisplayImage(
+                            new DefaultStreamedContent(new ByteArrayInputStream(event.getFile().getContents())));
```

Comment:
I think you can use different implementation of `StreamedContent` - `ByteArrayContent`. You won't need
double new operation